.github/CODEOWNERS

```
@@ -334,6 +334,8 @@ Makefile                                              @thiagokokada
 
 /modules/services/betterlockscreen.nix                @SebTM
 
+/modules/services/borgmatic.nix                       @DamienCassou
+
```

Can also add the test directory:

```suggestion
/modules/services/borgmatic.nix                       @DamienCassou
/tests/modules/services/borgmatic                     @DamienCassou

```